.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish cellxgene_schema_cli to PyPI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/all_ontology.json.gz'
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*.csv.gz'
+
+jobs:
+  publish-to-pypi:
+    name: Build and publish Python distributions to TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+      - name: setup git
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Bump PyPI release version
+        run: |
+          pip install bumpversion
+          pip install wheel
+          bumpversion --config-file .bumpversion.cfg minor --allow-dirty
+      - name: Build dist
+        run: >-
+          make pydist
+      - name: Publish distribution to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository-url: https://test.pypi.org/legacy/
+          packages-dir: cellxgene_schema_cli/dist/
+# TODO: Uncomment once ready to release to prod PyPI
```

Comment:
It feels odd to release to both pypi and testpypi at the same time. We shouldn't test that the RC works before release to pypi. We should install it using testpypi and run a few tests to confirm it works.

Reply:
Unit tests already run against a local build from the same dist files we push. What other tests do you have in mind?

Reply:
download from testpipy and make sure it runs correctly.